app_feup/android/app/src/main/res/drawable/launch_background.xml

```diff
@@ -7,9 +7,6 @@
        </shape>
     </item>
     <item>
-        <bitmap
-            android:gravity="center"
-            android:src="@mipmap/launch_image" />
+        <bitmap android:gravity="fill" android:src="@drawable/splash" />
```

Comment:
it looks like this `fill` is stretching the image (in my case testing on a 4" emulator it got squeezed vertically). I believe it would be better to crop it